### ISSUE TITLE
lock hint not provided after suspending

### ISSUE THREAD

**Issue by JezerM** [the-cavalry/light-locker#160]
LightDM offers the `get_lock_hint` method, to know whether lightdm is executed as a lockscreen or not.

When running the command `light-locker-command -l`, it works fine; the **lock hint** is provided, being set to "true". But after the system is suspended, **lock hint** is not provided, and so it is "false". I think this souldn't be this way, as it is still acting as a lock screen.

**Comment by jarnos** [comment]
But is system locked after resuming from suspend? Is the locking command run beforehand?

**Comment by JezerM** [comment]
Yes, locking works perfectly after resuming from suspend. The only problem is that **lock hint** is not provided.

I guess the locking command runs when suspend starts. When it resumes, LightDM resumes too (or starts).
